docs/config.rst

```diff
@@ -1103,3 +1103,64 @@ HTTPS_PROXY                     Defines the HTTPS proxy server. As of version ``
 FTP_PROXY                       Defines the FTP proxy server. Proxy authentication is supported providing the credentials in the proxy URL e.g. ``ftp://user:password@proxy-host.com:port``. FTP proxy support requires version ``21.06.0-edge`` or later.
 NO_PROXY                        Defines one or more host names that should not use the proxy server. Separate multiple names using a comma character.
 =============================== ================
+
+
+.. _config-feature-flags:
+
+Feature flags
+=============
+
+Some features can be enabled using the ``nextflow.enable`` and ``nextflow.preview`` flags. These flags are specified in the pipeline script rather than the configuration file,
+and they are generally used to introduce experimental or other opt-in features.
```

Comment:
Actually I'm not sure if these flags can be specified in the config file, @pditommaso can you confirm